apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java

```diff
@@ -241,4 +241,5 @@ public class ComponentsDefine {
 
     public static final OfficialComponent AEROSPIKE = new OfficialComponent(149, "Aerospike");
 
+    public static final OfficialComponent NACOS = new OfficialComponent(150, "Nacos");
```

Comment:
Please register the ID in the main repo. 
If you want to show the logo on the topo, you need to submit the logo to the UI repo as well.

Reply:
ok,i will summit the ID in the main repo

Reply:
According to https://github.com/apache/skywalking-booster-ui/pull/311#issuecomment-1680548512, this should be renamed as `nacos-client`.

Reply:
> According to [apache/skywalking-booster-ui#311 (comment)](https://github.com/apache/skywalking-booster-ui/pull/311#issuecomment-1680548512), this should be renamed as `nacos-client`.

shall i add nacos server component ?